secrets/store_test.go

```diff
@@ -37,12 +37,72 @@ const specificationExample = `
 	}
 }`
 
+var externalAccountKey = `
+{
+  "request_id": "1afc3036-2282-d483-c2d4-6d483efdf16c",
+  "lease_id": "",
+  "lease_duration": 2764800,
+  "renewable": false,
+  "data": {
+    "type": "versioned",
+    "current": "YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXowMTIzNDU=",
+    "previous": "aHVudGVyMg=="
+  },
+  "warnings": null
+}
+`
+
+var someAPIKey = `
+{
+  "request_id": "1afc3036-2282-d483-c2d4-6d483efdf16c",
+  "lease_id": "",
+  "lease_duration": 2764800,
+  "renewable": false,
+  "data": {
+    "type": "simple",
+    "value": "Y2RvVXhNMVdsTXJma3BDaHRGZ0dPYkVGSg==",
```

Comment:
Sanity check: this isn't a real secret, right? Looks more random than the rest.

Reply:
I dont know, I copied this from the other example secret that this test was already using

Reply:
I think it's a valid secret that's only used in unit tests. @pacejackson might know more of the origin of it 😅 

I'm 99% sure it's not something we used in prod.